### PR TITLE
Create manifest files for existing transform extensions

### PR DESCRIPTION
**Pull request by eslavich**
This PR creates manifest files for the existing transform extensions, whose versions each correspond to an ASDF Standard version.  A few questions to consider:

- Do we like including the title/description in the manifest file?  It's a good bit of content for one file.  If we want to merge schemas together, then examples will need to go into separate files and be linked to the tag entry by URI.  The alternative is to continue to maintain the one-schema-per-tag system.

- Is there a need for a separate "implementation description" property?  Long term I've been assuming that we'll use the title and description as display values to users in notebooks, and if we do that it may be nice to move some of the nitty gritty details into a third string property.

- I put the manifest ids and the extension URIs into `http://stsci.edu` URIs, under the "what happens in stsci.edu, stays in stsci.edu" principle.  An example:

```
id: http://stsci.edu/asdf/extensions/transform/manifests/transform-1.0.0
extension_uri: http://stsci.edu/asdf/extensions/transform-1.0.0
```

Are those reasonable choices?

### CODE REVIEW

File: tests/test_integration.py [hunk 1]
```diff
@@ -4,13 +4,36 @@
 import yaml
 
 
-def test_schemas():
-    schemas_root = Path(__file__).parent.parent / "resources" / "stsci.edu" / "schemas"
+def test_resources():
+    resources_root = Path(__file__).parent.parent / "resources"
```
Comment (perrygreenfield) [inline tests/test_integration.py:8]: I have to admit my ignorance of this new paradigm. I'm not sure I like the magical use of the / operator though. Maybe I will come to love it though.

File: scripts/generate_manifest.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import argparse
```
[the remark this answers is not in the document]
Comment (eslavich) [reply scripts/generate_manifest.py:1]: Oh yes definitely needs a docstring, I'll do that.

File: tests/test_integration.py (hunk 1 above)
Comment (eslavich) [reply tests/test_integration.py:8]: It's a little cutesy, isn't it...

**Comment by eslavich** [comment]
> Do we have a need yet? If not, can it not wait until we do?

Yea, good point, we should just wait and see how things develop.

> Is there an alternative you are considering?

The other option is locating them off asdf-format.org, but I don't think that makes sense given that everything else lives in stsci.edu.

**Pull request merged**